docs/developer/gui.ipynb

```diff
@@ -171,20 +171,12 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "nbsphinx": "hidden"
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
     "from ess.reduce.ui import WorkflowWidget\n",
     "\n",
-    "# Select Workflow\n",
-    "ess_widget.children[0].children[0].value = RandomDistributionWorkflow\n",
-    "# Set Output Parameter\n",
-    "body_widget: WorkflowWidget = ess_widget.children[1].children[0]\n",
-    "body_widget.output_selection_box.typical_outputs_widget.value = [Histogram]\n",
-    "body_widget.parameter_box.parameter_refresh_button.click()\n",
-    "body_widget.result_box.run_button.click()"
+    "ess_widget.children[0].children[0].value = RandomDistributionWorkflow"
```

Comment:
I think it's fine as a fix for now.
I don't want to spend time understanding the underlying issue with ipywidgets, threads, event loops and trailtets...